packages/js-utils/tsconfig.json

```diff
@@ -0,0 +1,35 @@
+{
+	"compilerOptions": {
+		"target": "ES5",
+		"lib": [ "DOM", "DOM.Iterable", "ESNext" ],
```

Comment:
Do we need those `DOM` libs for anything?

Reply:
I don't thinks so. Fixed in https://github.com/Automattic/wp-calypso/pull/50263/commits/31b8b3d2521bf99a7a3cb15677166f82c1b65317